tests/flow_function_parentheses/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -79,16 +113,12 @@ type f = (?arg) => void;
 
 class X {
   constructor(
-    ideConnectionFactory:
-      child_process$ChildProcess
-     => FlowIDEConnection = defaultIDEConnectionFactory
+    ideConnectionFactory: child_process$ChildProcess => FlowIDEConnection = defaultIDEConnectionFactory
```

Comment:
This no longer breaks but honestly, the way it was breaking before was super confusing.